ecl/hqlcpp/hqlcpp.cpp

```diff
@@ -1946,6 +1946,8 @@ void HqlCppTranslator::cacheOptions()
         }
     }
 
+    updateTraceFlags(loadTraceFlags(wu(), eclccTraceOptions, queryTraceFlags()));
```

Comment:
There is a options.traceAll Debug option (line 1921 of this file).  Should loadTraceFlags be making use of this options.traceAll?